### PR TITLE
project-guess-who-shaun-newmark

### PR DESCRIPTION
**Pull request by newmarkdev**

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -262,82 +273,135 @@ const selectQuestion = () => {
...
 }
 
-// It'll filter the characters array and redraw the game board.
+// Filter the characters array and redraw the game board.
 const filterCharacters = (keep) => {
-  // Show the correct alert message for different categories
-  if (group === 'accessories') {
+
+  // Show alert messages for different categories
+  const { attribute, category, value } = currentQuestion
```
Comment (TheErry) [inline code/script.js:317]: I don't think it should be spaces between the brackets there

File: code/script.js [hunk 2]
```diff
@@ -262,82 +273,135 @@ const selectQuestion = () => {
...
+let currentScore = 0 
+const checkScore = (suspect) => {
+  suspect === secret.name ? currentScore++ : currentScore--
+}
+const scoreBoard = () => {
+  let playerScore = currentScore
+  return scoreBoardText.innerHTML = `Your score is: ${playerScore}`
+}
+
+
```
Comment (TheErry) [inline code/script.js:383]: I like that you added a score board! I know that you said yourself that it wasn't working correctly (giving negative scores and such). I saw it still does that in the game. Unforunately I can't really help you with that, I can't see what's causing it either...

File: code/style.css [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
   display: flex;
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
+  font-family: "Montserrat", sans-serif;
+  background: linear-gradient(
+    189deg,
+    rgba(166, 228, 255, 1) 0%,
+    rgba(89, 138, 255, 1) 59%
+  );
```
Comment (TheErry) [inline code/style.css:16]: This is cool stuff! Nice gradient!

File: code/style.css [hunk 4]
```diff
@@ -34,7 +50,7 @@ select {
   display: block;
   border: none;
   font-size: 22px;
-  font-family: 'Montserrat';
+  font-family: "Montserrat";
```
Comment (TheErry) [inline code/style.css:53]: Neat!

File: code/index.html [hunk 5]
```diff
@@ -1,64 +1,64 @@
...
-      <img
-        class="guess-who-icon"
-        src="images/guess-who-bubble.png"
-        alt="Guess Who"
-      />
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Guess Who | Game</title>
+  <link rel="stylesheet" href="style.css" />
```
Comment (TheErry) [inline code/index.html:8]: I read somewhere that we should write like ./style.css in refs, but I can't really find it where it said that now however. I got a "good job for remembering this" comment in my last project though :woman_shrugging:  Sneaky of them not to do that in the pre-written code! :P

**Review by TheErry — APPROVED** [review]
Great job! I'm a bit unsure about how importnant it is to add the other properties in the options section, but since it wasn't really in the requirenments I guess I can set this as approved :)

File: code/index.html [hunk 6]
```diff
@@ -1,64 +1,64 @@
...
+        <!-- Add the rest of eye colors as options  -->
+      </optgroup>
+      <optgroup label="accessories">
+        <option value="glasses">glasses</option>
+        <!-- Add the other accessory option here. (hat)  -->
+      </optgroup>
+      <optgroup label="other">
+        <option value="smoker">a smoking habit</option>
+      </optgroup>
+    </select>
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/index.html:39]: This is a very important thing that you add all of the combinations to your options list. Please do it 🤓

File: code/script.js [hunk 7]
```diff
@@ -262,82 +273,135 @@ const selectQuestion = () => {
...
 }
 
-// This function should be invoked when you click on 'Find Out'.
+// Check secret character object on click of 'Find Out'.
 const checkQuestion = () => {
+  const keep = currentQuestion.value === secret[currentQuestion.attribute]
+  filterCharacters(keep)
   // Compare the currentQuestion with the secret person.
   // See if we should keep or remove people based on that
   // Then invoke filterCharacters
```
Comment (KarolCislowski) [inline code/script.js:310]: It would be good if you clean-up a little bit old comments which ware here to guide you 😸

File: code/script.js (hunk 1 above)
Comment (KarolCislowski) [reply code/script.js:317]: It is ok 👍🏻

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
It is a very nice project 💯 but you need to add more options to filter your board, top make this game possible to win 🥇

**Review by KarolCislowski — APPROVED** [review]
Good job 😸

**Pull request closed without merging**